### PR TITLE
Add covid-19 export records

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#9599](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9599)

* [X] Enhancement
* [ ] Bug
* [ ] Documentation

## Description
- Add covid-19 export records option.
- Add covid-19 export records pdf feature.
- Add func. tests.
image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [X] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES

### Browsers Tested

* [X] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

**Comment by tiagograf-AOT** [comment]
It seems like there something wrong with the dates, can you confirm?

**Review by sslaws — DISMISSED** [review]

**Review by tiagograf-AOT — CHANGES_REQUESTED** [review]
possible date problem?

File: Apps/WebClient/src/ClientApp/src/components/report/covid19.vue [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                );
+            })
+            .finally(() => {
+                this.isLoading = false;
+            });
+    }
+
+    private sortEntries() {
+        this.records.sort((a, b) =>
+            a.messageDateTime > b.messageDateTime
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/ClientApp/src/components/report/covid19.vue:79]: sort must be wrong too!

**Review by sslaws — DISMISSED** [review]

File: Apps/WebClient/src/ClientApp/src/components/report/covid19.vue [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                    <b-col class="col">Test Type</b-col>
+                    <b-col class="col">Test Location</b-col>
+                    <b-col class="col">Result</b-col>
+                </b-row>
+                <b-row v-for="item in records" :key="item.id" class="item py-1">
+                    <b-col
+                        data-testid="covid19ItemDate"
+                        class="col my-auto text-nowrap"
+                    >
+                        {{ formatDate(item.collectedDateTime) }}
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/ClientApp/src/components/report/covid19.vue:194]: it should be item.labResults[0].collectedDateTime

**Review by tiagograf-AOT — CHANGES_REQUESTED** [review]
commented a bug with the date

File: Apps/WebClient/src/ClientApp/src/components/report/covid19.vue (hunk 2 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/components/report/covid19.vue]: fixed. thanks @tiagograf-AOT

**Review by tiagograf-AOT — APPROVED** [review]

**Review by sslaws — APPROVED** [review]

**Pull request merged**